### PR TITLE
Add Vue integration with Django

### PR DESCRIPTION
**Pull request by disusered**
- Rename dashbord to dashboard
- Add ES6 via Babel
- Add automated code formatting and linting
- Add NodeJS buildpack for Heroku for front end
- Add Vagrant config for front end
- Add Webpack integration with Django
- Fix post-deploy script for Heroku
- Add production minification
- Add Vue
- Scope to dashboard page
- Update README
- Hook up Websocket with Vue for reactive front end

### CODE REVIEW

**Comment by disusered** [comment]
Updated:

- Add new javascript entry point
- Add Vue Modal plugin
- Add global Vue controller for modal
- Add global Vue controller for nav
- Replace links to bootstrap modal with links to Vue modal
- Render modal content in Vue modal
- Setup common chunks to de-duplicate controller code
- Add UI-kit tab switcher to modal
- Add UI-kit sub-header to modal
- Update form and button styles in modal
- Use dashboard header for incidents view
- Update table styles for incidents view

File: ersteops/templates/emergency/list.html [hunk 1]
```diff
@@ -1,44 +1,60 @@
-{% extends 'base.html' %}
+{% extends 'home/base_ui.html' %}
 {% load staticfiles %}
 
-{% block content %}
-<h1>Incidents</h1>
+{% block header %}
```
Comment (disusered) [inline ersteops/templates/emergency/list.html]: Queria agregar el titulo de la vista en vez de usar la que esta hard-coded en el template. Busque instrucciones y recomendaban moverle al modelo de Python y/o crear templatetags, es mucho rollo?

Comment (xtornasol512) [reply ersteops/templates/emergency/list.html]: Ok puedes explicar que titulo de la vista  te refieres?

**Comment by disusered** [comment]
@xtornasol512 ya debe funcionar en Heroku por si quieres checar/probar!

**Comment by xtornasol512** [comment]
Solo hay unos checks en el dashboard que debo comprobar y ya le hago merge!

**Pull request merged**